### ISSUE TITLE
software icon remaining on the taskbar after start

### ISSUE THREAD

**Issue by wisdomizer** [rocksdanister/lively#36]
Hello to you all!

First, I want to thank you so much for the wonderful tool you have created!
Very much what I wanted, though a few things could be added :)

I have one "problem" though (sorry for the repetition),
Either after I start the program after the very PC is started, or it is put on automatic start, the icon itself remains on the taskbar.   After I close it, the desktop freezes, and I have to go inside the very program to refresh and/or reselect the video I want it to be played. Somewhat frustrating.

I really hope you will be able to help me out here and resolve it, as I see you have done an amazing job with the program itself

Greetings
Anton

**Comment by rocksdanister** [comment]
> the icon itself remains on the taskbar. 

Lively needs to run in the background, so yes icon will remain in taskbar.

> After I close it, the desktop freezes
Close how? desktop freezes what does that mean?

**Comment by wisdomizer** [comment]
> > the icon itself remains on the taskbar.
> 
> Lively needs to run in the background, so yes icon will remain in taskbar.
> 
> > After I close it, the desktop freezes
> > Close how? desktop freezes what does that mean?

The icon being on the tray it is needed, indeed; but I am talking about the big icons that pile up on the task bar, much like when you open any other window or application.

After I close  it (the taskbar icon) the very video that is playing, stops it, it freezes, and I have to click on the "tray icon" in order to reselect the video to start playing again. that is all.

**Comment by rocksdanister** [comment]
what is your windows version?

**Comment by wisdomizer** [comment]
windows 10

**Comment by rocksdanister** [comment]
Does it happen with web type wallpapers?

**Comment by wisdomizer** [comment]
i only use it for videos that I have on my computer

**Comment by rocksdanister** [comment]
I'm not asking what you like/use, I'm asking you to test something for me :/

**Comment by wisdomizer** [comment]
what is a web type of wallpaper - like to put wallpaper on your screen directly from the web?

**Comment by rocksdanister** [comment]
web pages, when you select a wallpaper on rightside it will be written Type: Web
Lively includes some webtype wallpapers: Waves, Fluid etc

**Comment by wisdomizer** [comment]
I cannot do it because it says I do not have the full version

and btw, you did not have to be rude

**Comment by rocksdanister** [comment]
>I cannot do it because it says I do not have the full version

I stopped providing lite version, if you are from v0.6.5 onwards, meaning either your antivirus software deleted the browser plugin or something else went wrong.
> and btw, you did not have to be rude

I'm not a native English speaker and I mostly ask direct questions when trouble shooting problems... and then you gave a completely unrelated answer.. ofc I will get annoyed.

**Comment by wisdomizer** [comment]
can you direct me to someone else who can help me?

not being native to any language, does not mean you are allowed to be rude;
everyone can get annoyed, but you are the support, and I really hope you are human

so, please, either transfer me to someone more human, or at least tell me which name/category to look for this question.

**Comment by rocksdanister** [comment]
you do realize this is a free project being run by a single person? 
I don't owe you anything, if you don't like it don't use it.

**Comment by wisdomizer** [comment]
and once again, you are being rude, instead of being pleasant (or at least try to)

when a person says "I am not asking what...." you are the definittion of being rude. Ask anyone this question and see how their first reaction will be

you could have said "I am trying to test something. I can you do me a favor"  - see? be pleasant.

try this behavior you are showing with me with any of you relatives and see the result.

no, you not owe me anything - but at least I was nice unlike you
and if you had read my very first message, I expressed big gratitude for such a wodnerful tool, and I was merely asking for assistance and resolution.

bottom line, show some human attitude
after all you are speaking with people and not with machines.

also, I hope you read my last response to your unkind request - it does not allow me to do it

and finally, you mentioned this is being run by a single person?
mind if I ask is this person you or someone else?

and might want to start using thank, nice day and similar things
